### DISCUSSION TITLE
Think about important metrics for robotoff quality monitoring

### DISCUSSION THREAD

**Discussion by alexgarel** [openfoodfacts/robotoff#1226] [Ideas]
What are the important metrics to be able to measure the volume and quality of robotoff work ?

**Comment by alexgarel** [comment]
One type of monitoring can be targeted : eg. how many agrybalise category do we predict / we don't predict.

**Comment by raphael0202** [comment]
A must-have metrics to keep track of insight table current state:
number of insights grouped by (type, annotation, automatic_processing, predictor, reserved_barcode).
